sdk/src/CrashReporter.ts

```diff
@@ -43,6 +43,10 @@ export default class CrashReporter {
   private onBeforeSendingCrashReport: BeforeSendHandler | null;
   private raygunCrashReportEndpoint = 'https://api.raygun.com/entries';
 
+  private local_storage_key : string = "raygun4reactnative_local_storage";
+  private readonly RAYGUN_RATE_LIMITING_STATUS_CODE : number = 429;
+  private maxLocallyStoredCrashReports : number = 32;
```

Comment:
Does this need to be bumped to 64 by default?

Reply:
DONEZO